crypto/onetimesig.go

```diff
@@ -41,8 +41,9 @@ type OneTimeSignature struct {
 	PK  ed25519PublicKey `codec:"p"`
 
 	// Old-style signature that does not use proper domain separation.
-	// PKSigOld is a signature of (PK || BatchID) under the master key (OneTimeSignatureVerifier).
-	// TODO(adam): This should be safe to delete now, right?
+	// PKSigOld is unused; however, unfortunately we forgot to mark it
+	// `codec:omitempty` and so it appears (with zero value) in certs.
+	// This means we can't delete the field without breaking catchup.
 	PKSigOld ed25519Signature `codec:"ps"`
```

Comment:
@zeldovich Out of scope for this PR, but I was thinking of one way we can deprecate this field: Since signatures are never committed to in any way (they're part of the cert but they themselves are not signed), we could update all archival servers to supply blocks with this field missing. After all archives have updated, it should be safe to drop the field.